core/src/main/java/org/apache/calcite/plan/SubstitutionVisitor.java

```diff
@@ -115,8 +115,11 @@
  * {@link org.apache.calcite.rel.core.TableScan},
  * {@link org.apache.calcite.rel.core.Filter},
  * {@link org.apache.calcite.rel.core.Project},
+ * {@link org.apache.calcite.rel.core.Calc},
  * {@link org.apache.calcite.rel.core.Join},
  * {@link org.apache.calcite.rel.core.Union},
+ * {@link org.apache.calcite.rel.core.Intersect},
+ * {@link org.apache.calcite.rel.core.Sort},
```

Comment:
Doc like enumerating the classes are hard to maintain, people should always remember to update the references, let's avoid to make such kind of doc.

Reply:
Thanks for you review.